### PR TITLE
Financial instruments transformer

### PR DESCRIPTION
**Pull request by danielcopaciu**
https://github.com/Financial-Times/financial-instruments-transformer
https://github.com/Financial-Times/financial-instruments-transformer/pull/1

### CODE REVIEW

**Comment by BerniVarga** [comment]
+1

File: services.yaml [hunk 1]
```diff
@@ -61,6 +61,11 @@ services:
   version: v4.0.0
 - name: elb-presence@.service
   count: 2
+- name: financial-instruments-transformer-sidekick@.service
+  count: 1
+- name: financial-instruments-transformer@.service
+  version: v0.0.0-alpha.2
```
Comment (tamas-molnar) [inline services.yaml]: Promoting alpha version to prod?

**Comment by Nesze** [comment]
Tested in XP.

**Comment by Nesze** [comment]
+1

**Comment by danielcopaciu** [comment]
Version 0.0.1 tested successfully in xp

**Pull request merged**